pages/healingShop.php

```diff
@@ -48,13 +48,15 @@ function healingShop() {
 			if(!isset($ramen_choices[$heal])) {
 				throw new RuntimeException("Invalid choice!");
 			}
-			if($player->getMoney() < $ramen_choices[$heal]['cost']) {
+			if($player->getMoney() < $ramen_choices[$heal]['cost'] && !$system->isDevEnvironment()) {
```

Comment:
Maybe we could make this simpler by changing the costs above? e.g.


```php
if($system->isDevEnvironment()) {
   $ramen_choices['vegetable']['cost'] = 0;
   $ramen_choices['pork']['cost'] = 0;
   $ramen_choices['deluxe']['cost'] = 0;
}
```

I think this way it would also reflect in the displayed costs

Reply:
Cleaner, changing that now.